package.json

```diff
@@ -1,11 +1,18 @@
 {
-  "name": "@vtex/cli-plugin-template",
-  "description": "vtex plugin template",
-  "version": "0.0.0",
-  "bugs": "https://github.com/vtex/cli-plugin-template/issues",
+  "name": "@vtex/cli-plugin-redirects",
+  "description": "vtex plugin redirects",
+  "version": "0.0.1-beta",
+  "bugs": "https://github.com/vtex/cli-plugin-redirects/issues",
   "dependencies": {
     "@oclif/command": "^1",
     "@oclif/config": "^1",
+    "@tiagonapoli/oclif-plugin-spaced-commands": "^0.0.6",
+    "@types/fs-extra": "^9.0.1",
+    "@types/json2csv": "^5.0.1",
+    "@types/progress": "^2.0.3",
+    "@types/ramda": "^0.27.14",
+    "json-array-split": "^1.0.0",
```

Comment:
Maybe this and the next import is exclusive from this plugin, checkout in `toolbelt` to see if can be removed

Reply:
Yes, I removed in this [commit](https://github.com/vtex/toolbelt/commit/3ea80cbea22251503d9095b52de7e8dc3f60befb) the json2csv and progress, but I think I forgot the json-array-split, thanks for the heads up!